web/react/components/edit_channel_purpose_modal.jsx

```diff
@@ -90,7 +90,7 @@ export default class EditChannelPurposeModal extends React.Component {
                     </Modal.Title>
                 </Modal.Header>
                 <Modal.Body>
-                    <p>{`Describe how this ${Utils.getChannelTerm(this.props.channel.channelType)} should be used.`}</p>
+                    <p>{`Describe how this ${Utils.getChannelTerm(this.props.channel.channelType)} should be used. This text appears in the channel list in the "More..." menu and helps other decide whether to join.`}</p>
```

Comment:
Currently says `helps other decide` should be `helps others decide`.
Also I think the clarification of the location of the channel list is unnecessary as we in the future may change the name or location of the "More..." menu and there may be other places where this description is used.